spotbugs/src/main/java/edu/umd/cs/findbugs/ba/DefaultNullnessAnnotations.java

```diff
@@ -431,6 +431,10 @@ public static void addDefaultNullnessAnnotations(INullnessAnnotationDatabase dat
         database.addMethodAnnotation("java.util.Objects", "requireNonNull", "(Ljava/lang/Object;Ljava/lang/String;)Ljava/lang/Object;",
                 true, NullnessAnnotation.NONNULL);
 
+        database.addMethodParameterAnnotation("kotlin.jvm.internal.Intrinsics", "checkNotNullParameter", "(Ljava/lang/Object;)V",
+                true, 0, NullnessAnnotation.NONNULL);
```

Comment:
I couldn't find this function in the current master of [kotlin](https://github.com/JetBrains/kotlin/blob/master/libraries/stdlib/jvm/runtime/kotlin/jvm/internal/Intrinsics.java#L129). Is it in an earlier version?